### DISCUSSION TITLE
is this possible to override receipt (PDF)?

### DISCUSSION THREAD

**Discussion by ronakabhattrz** [pay-rails/pay#445] [General]
@excid3 
is this possible to override the receipt (PDF) which we are sending after payment success?

Need to add more details in this PDF so how can I?
<img width="800" alt="Screenshot 2021-09-29 at 12 25 18 AM" src="https://user-images.githubusercontent.com/21212187/135148723-209c9437-a74d-425b-8a2d-5596e6b1e3f9.png">

**Comment by excid3** [comment]
You'd just override these methods as needed on `Pay::Charge`: https://github.com/pay-rails/pay/blob/master/lib/pay/receipts.rb#L16

Documented how to override methods here: https://github.com/pay-rails/pay/blob/master/docs/8_customizing_models.md

👍
